### DISCUSSION TITLE
Webhook for new lisitngs and listings status change

### DISCUSSION THREAD

**Discussion by KushGoyal** [etsy/open-api#825] [Q&A]
I am using etsy API to enable sellers to manage inventory for listings. A major issue I face is when new listings are created. My software needs to detect the new listing as soon as it is created. The same goes for status change of listing. Without any webhooks I have to run cron job. Which is not ideal given the throttling limits.

Is there any way I can subscribe to an event which can fire when a new listing is made or the listing status has changed.

Thanks.

**Comment by Gareth064** [comment]
Unfortunately the Etsy API has no concept of webhooks or events.

This has been in discussion for a while now and is being "tracked" on. I would throw your support for it onto this issue #28
